(Reviewed) Proofreading beginning tutorial

John-hash-tech opened the pull request:
I made some changes to non-dialog text to improve the clarity of the English during the tutorial. See individual commit messages for an explanation of why I made each change.

John-hash-tech committed:
Update 70001.txt

In American English, it is a convention that commas and most other punctuation be enclosed within parentheses. I believe this is not the case in UK English. Feel free to disregard depending on the intended target audience for the translation.

John-hash-tech committed:
Update 70001.txt

The contents of these parentheses are slightly ambiguous. I specified that the Level is what must be 7+. I then replaced the plus sign with the phrase "or more" to maintain consistency with the "Level of 5 or more" earlier in the sentence.

John-hash-tech committed:
Update 70001.txt

Explicitly stated the term Life Points before introducing an abbreviation (LP), as was done in the original Japanese text.

Added a sentence explaining how to reduce LP in order to accommodate the contrasting "However" at the start of the next sentence, which explains how to prevent the loss of LP.

Added a comma, since "as long as" marks the beginning of the sentence as a dependent clause. Clauses are separated by a comma when the dependent one comes first.

Removed the last sentence because it is redundant (it restates the previous sentence) and ambiguous (Which player has one, and what does that player have?).

John-hash-tech committed:
Update 70001.txt

I believe this line of tutorial text is meant to emphasize the fact that the player can attack, not the fact that the attacks can be direct. It was probably written under the assumption the player would already be familiar with Yugioh, where monsters do all the attacking, which means this line is drawing attention to an important difference in game mechanics.

I added a mention of monsters to the last sentence to make it clearer who the point of reference is for the comparison.

HotHeadEmperor commented on CSV/Language/English/Card/70001.txt:
```diff
@@ -15,16 +15,16 @@ default
 {使用カード}を召喚！,I summon {使用カード}!
 チュートリアル２：召喚,Tutorial 2: Normal Summoning
-プレイヤーは手札から１ターンに１度だけモンスターを召喚することができます。場に出したモンスターはHPがゼロになると破壊され墓地へ送られます。レベルが５以上のモンスターは場のモンスターをコストに支払わないと召喚することができません（レベルが５～６の場合１体、レベルが７以上の場合２体）。,Players may normal summon 1 monster each turn from their hands. A monster is destroyed and sent to the Graveyard when their HP reaches 0. Monsters with a Level of 5 or more require monsters to be tributed from the field to be played. (1 monster for a Level ...
+プレイヤーは手札から１ターンに１度だけモンスターを召喚することができます。場に出したモンスターはHPがゼロになると破壊され墓地へ送られます。レベルが５以上のモンスターは場のモンスターをコストに支払わないと召喚することができません（レベルが５～６の場合１体、レベルが７以上の場合２体）。,Players may normal summon 1 monster each turn from their hands. A monster is destroyed and sent to the Graveyard when their HP reaches 0. Monsters with a Level of 5 or more require monsters to be tributed from the field to be played (1 monster for a Level  ...
```
Good catch,  but I think we could make it a little smoother by changing 'more' to 'higher'

Maybe '1 tribute for monsters level 5-6, and 2 tributes for monsters level 7 and higher.'

John-hash-tech replied:
I also prefer the change to "higher." I only left it as "more" because I didn't want to trample over the translator's stylistic choice.

I also prefer the change from "1 monster for" to something like "1 tribute monster for."

In other words, I fully agree.

HotHeadEmperor commented on CSV/Language/English/Card/70001.txt:
```diff
@@ -15,16 +15,16 @@ default
-先に相手のライフポイント(LP)をゼロにした方が勝者です。ライフを減らすためには相手プレイヤーに攻撃をする必要があります。ただし場に相手のモンスターがいる場合は、相手プレイヤーに直接攻撃できないので、先にモンスターを倒さなければなりません。,A player wins when they reduce their opponent's LP to 0. However; as long as a player has one or more monsters on the field they may not be attacked directly. Players are unable to directly attack their opponent as long as they have one.
-このゲームではプレイヤー自身も相手に攻撃を行うことができます。プレイヤーの攻撃力は比較的高いので、積極的に攻撃をしかけていくことが重要です。,In this game; players may attack their opponents directly. The attack power of the player is comparatively higher; so attacking aggressively is key.
+先に相手のライフポイント(LP)をゼロにした方が勝者です。ライフを減らすためには相手プレイヤーに攻撃をする必要があります。ただし場に相手のモンスターがいる場合は、相手プレイヤーに直接攻撃できないので、先にモンスターを倒さなければなりません。,A player wins when they reduce their opponent's Life Points (LP) to 0. You can reduce your opponent's LP by attacking them directly. However; as long as a player has one or more monsters on the field; they may not be attacked directly.
+このゲームではプレイヤー自身も相手に攻撃を行うことができます。プレイヤーの攻撃力は比較的高いので、積極的に攻撃をしかけていくことが重要です。,In this game; the players themselved may attack their opponents. Compared to monsters; the attack power of the players tends to be higher; so attacking aggressively is key.
```
'players themselves' and maybe 'are able to' instead of 'may' but thats more of a preference thought

John-hash-tech committed:
Update 70001.txt

PR feedback:
wording changed
typo fixed

John-hash-tech commented:
I have updated this branch based on PR feedback.

HotHeadEmperor commented on CSV/Language/English/Card/70001.txt:
```diff
@@ -15,16 +15,16 @@ default
-先に相手のライフポイント(LP)をゼロにした方が勝者です。ライフを減らすためには相手プレイヤーに攻撃をする必要があります。ただし場に相手のモンスターがいる場合は、相手プレイヤーに直接攻撃できないので、先にモンスターを倒さなければなりません。,A player wins when they reduce their opponent's LP to 0. However; as long as a player has one or more monsters on the field they may not be attacked directly. Players are unable to directly attack their opponent as long as they have one.
-このゲームではプレイヤー自身も相手に攻撃を行うことができます。プレイヤーの攻撃力は比較的高いので、積極的に攻撃をしかけていくことが重要です。,In this game; players may attack their opponents directly. The attack power of the player is comparatively higher; so attacking aggressively is key.
+先に相手のライフポイント(LP)をゼロにした方が勝者です。ライフを減らすためには相手プレイヤーに攻撃をする必要があります。ただし場に相手のモンスターがいる場合は、相手プレイヤーに直接攻撃できないので、先にモンスターを倒さなければなりません。,A player wins when they reduce their opponent's Life Points (LP) to 0. You can reduce your opponent's LP by attacking them directly. However; as long as a player has one or more monsters on the field; they may not be attacked directly.
+このゲームではプレイヤー自身も相手に攻撃を行うことができます。プレイヤーの攻撃力は比較的高いので、積極的に攻撃をしかけていくことが重要です。,In this game; the players themselves able to attack their opponents. Compared to monsters; the attack power of the players tends to be higher; so attacking aggressively is key.
```
'Players themselves **are** able'

John-hash-tech committed:
Update 70001.txt

Added missing word back in.

John-hash-tech committed:
Tutorial punctuation

I removed the period because tildes are used consistently as sentence-ending punctuation already, making it redundant. (Note: lots of sentences end in hearts, which are accompanied by separate punctuation marks. Consequently, I do not consider hearts sentence-ending punctuation marks.)

I added commas to separate the clause out of the middle of the sentence. Em-dashes would probably be fine here as well, and I have noticed them used occasionally in this project, but I'm playing it safe with commas here.

John-hash-tech committed:
Update 70002.txt

I think this is a better word choice, but this change is not essential.

John-hash-tech committed:
Beginning Tutorial ellipsis

Removed extra dots where sentences ending with ellipsis had four dots. I'm guessing the ellipsis characters were changed with find-and-replace, then the Japanese period characters were changed with find-and-replace, resulting in four dots.

John-hash-tech committed:
Update 70002.txt

I don't think anyone I know would use the phrase "rush into things absentmindedly." "Rush" implies enthusiasm while "absentmindedly" implies detachedness, so I think this word fits better here, but this change is not essential.

John-hash-tech committed:
Update 10001.txt

Mostly missing words and spaces from the succubus maid monster in the tutorial.

John-hash-tech committed:
Update 10003.txt

Mostly missing words and spaces from the advanced samurai monster in the tutorial.

John-hash-tech committed:
Update 70002.txt

Mostly small changes to Master Konoha's lines in the tutorial. I changed line 95 slightly because the last sentence sounded redundant.

Giratena commented:
Making this as ready for review as I assume this is the case based on discussion in Discord ^^

Giratena commented on CSV/Language/English/Card/10001.txt:
```diff
@@ -19,10 +19,10 @@ EU1_Maid_Cum,
 この気持ちよさの前なら、\nバトルなんてどっちだっていいですよね？くすくす❤, It's so easy to forget about the battle when your cock is feeling good; isn't it? Ehehe.❤
 お客様に最上のご奉仕を❤\nほら、お胸で挟んでさしあげますね❤,Sir; please allow me to service you.❤ Hora; I'll squeeze your penis between my breasts.❤
-どれだけ強いモンスターでも、サキュバスの快楽の前では無力ですわ❤,All monsters no matter how strong are rendered powerless when pleasured by a succubus.❤
+どれだけ強いモンスターでも、サキュバスの快楽の前では無力ですわ❤,All monsters; no matter how strong; are rendered powerless when pleasured by a succubus.❤
```
Hmm what if we use an Em dash here to avoid the double comma?

`All monsters—no matter how strong—are rendered powerless when pleasured by a succubus.❤`

HotHeadEmperor replied:
I think commas make more sense here. Em dashes are usually used for extra information more tangential to the sentence, where as this seems like the purpose of the sentence. If we rearranged the sentence to something like "No matter how strong they are, all monsters are rendered powerless when pleasured by a succubus' the sentence still makes sense, so I think commas are the right punctuation.

Either way, make sure you're remembering the spaces -before the and after the last- when you use em dashes.

Giratena replied:
Ok sounds good.  I'd be happy with a single comma too ^^

John-hash-tech replied:
I would rather use em dashes than keep just one of the added commas. How do we feel about a rearrangement like this?
`No matter how strong; all monsters are rendered powerless when pleasured by a succubus.❤`
or
`No matter how strong; any monster would be rendered powerless when pleasured by a succubus.❤`

HotHeadEmperor replied:
That's sounds good to me

Giratena replied:
Yep looks good to me!

Giratena commented on CSV/Language/English/Card/10001.txt:
```diff
@@ -57,14 +57,14 @@ EU1_Maid_Cum,
 そのまま力を抜いて、おっぱいに負けちゃっていいですよ。\nお客様❤,Sir; please go ahead and relax.❤ It's okay to lose yourself in my breasts.❤
 あら？少し挟まれただけで、イキそうなんですか？,Oh my. Are you already near orgasm after only a few squeezes?
-次のターンは攻撃しないでじっとしててくださいね❤\n谷間の中で、気持ちよく果てさせてあげます❤,Please don't do anything on your next turn; and I'll grant you an blissful climax in my breasts.❤
+次のターンは攻撃しないでじっとしててくださいね❤\n谷間の中で、気持ちよく果てさせてあげます❤,Please don't do anything on your next turn; and I'll grant you a blissful climax in my breasts.❤
```
Good change! I would also add:

In my breasts -> between my breasts

John-hash-tech replied:
I like the sound of that. There are actually a lot of "in my breasts" I could change in this file, but I think I want to keep the one in line 58. Thoughts?

Giratena replied:
Yes please, you're more than welcome to change them all! I think between is more natrual than "in" in most cases.

Giratena commented on CSV/Language/English/Card/10003.txt:
```diff
@@ -72,24 +72,24 @@ EU3_paizuriAnime,
 さっきは途中でまででしたけど、\n今度は逝くまでしてあげますよ❤,I stopped halfway through earlier; but this time I'll keep doing it until you cum~❤
 ねえ、さっきわたしがパフパフするの、\n見てましたよね。,Hey~ You saw me giving out a Puff-Puff earlier; didn't you?
-この胸で顔を挟んで、\n何度もぱふぱふ❤ぱふぱふ❤,My breasts pushed up against their face...rubbing them in blissful softness...bouncing them around...❤
+この胸で顔を挟んで、\n何度もぱふぱふ❤ぱふぱふ❤,My breasts pushed up against their face... rubbing them in blissful softness... bouncing them around...❤
```
Hmm I actuallly think the use of eclipses was grammatically fine to begin with here!

Not having a space makes it a continuous stream of ideas, which is more of the vibe we're going for.

For learnings sale, if we do have a space after a ..., we would also need to capitalize the next letter, as it marks the start of a new sentence.

HotHeadEmperor replied:
Every bit of punctuation needs a space. You only need to capitalize if the thought changes after the ellipses but you always put a space.

Giratena replied:
Discussed on discord and we will keep the spaces!

Giratena commented on CSV/Language/English/Card/10003.txt:
```diff
@@ -72,24 +72,24 @@ EU3_paizuriAnime,
 さあ、仲間を裏切って気持ちよくなることを選んだご褒美に\nたっぷり甘やかしてあげます❤,Now then; as a reward for choosing pleasure and betraying your allies; I'll be sure to spoil you lots.❤
 ほら、ぱふぱふ❤,Hora- Puff-Puff~❤
-はひぃっ、ああっ・・・❤,Owaah...ahh...❤
+はひぃっ、ああっ・・・❤,Owaah... ahh...❤
```
Ditto for this line here. Either remove the space or capitalize the A in Ahh

Giratena commented on CSV/Language/English/Card/10003.txt:
```diff
@@ -99,12 +99,12 @@ EU3_paizuriAnime,
 あひぃっ…❤❤あっ、ああっ……❤❤❤,Ahiee...❤❤ Ahh; ahhh...❤❤❤
 ふふ、いいですよ❤\nわたしの体に体重あずけながら、\nどぴゅどぴゅって出しちゃってください。,Fufu. That's it.❤ Go ahead and rest the weight of your body on me as you let your balls empty out~
-とどめに、\nおっぱいぎゅーってしてあげます。\nほら、ぎゅー・・・❤,And for the finishing blow... I'll squeeze my boobs tightly.\n. Ready? Squeezeee~❤
+とどめに、\nおっぱいぎゅーってしてあげます。\nほら、ぎゅー・・・❤,And for the finishing blow... I'll squeeze my boobs tightly.\n Ready? Squeezeee~❤
```
Good spot!

We can also remove the /n here as we aren't using them in translated dialogue.

John-hash-tech replied:
There are also a bunch of \n in the English part of this file. I'm going to have to retest this encounter once I get rid of them.
Edit: had to use regex to make sure I found them all lmao

HotHeadEmperor replied:
Yeah I'm general were getting rid of all '/n's in the english. The game wraps text pretty well itself. As long as they're in the Japanese it renders fine

Giratena commented on CSV/Language/English/Card/10003.txt:
```diff
@@ -99,12 +99,12 @@ EU3_paizuriAnime,
 結局指一本触られていないのに、\nおちんちんからびゅーびゅー出しちゃいましたね❤,I haven't laid a finger on you; and yet your penis can't help but empty its load.❤
 勝てる戦いをこんなカッコ悪い射精で棒に振って、\n良かったんですか？,Was it a good idea to sacrifice a winning battle so you could ejaculate in such an unsightly way?
-まあ、わたしのおっぱいで\nそんな幸せそうな顔をしているあなたには\n聞くまでもないかもしれませんけど❤くすくす❤,Well...if my boobs give you such a happy expression; then I guess I don't need to ask you to know the answer.
+まあ、わたしのおっぱいで\nそんな幸せそうな顔をしているあなたには\n聞くまでもないかもしれませんけど❤くすくす❤,Well... if my boobs give you such a happy expression; then I guess I don't need to ask you to know the answer.
```
Remove space or capitalize I here

Giratena commented on CSV/Language/English/Card/10003.txt:
```diff
@@ -119,19 +119,19 @@ EU3_paizuriAnime,
 自分からおっぱいに負けちゃうのはどうでしたか？\n変態モンスターさん❤,How did you find losing your own will to my boobs?\nMr. Pervert Monster-❤,
 あらあら❤あなたもおっぱいに負けたくなっちゃったんですか？,Oh my.❤ Do you want to lose to my boobs as well?
-自分から屈服に来れるだなんて、\n素直でいい子ですね。\nもちろん見せてあげますよ。ほら…❤,Coming over to surrender of your own volition...what a good obedient boy you are.\nI'd be happy to show you. Now then...❤
+自分から屈服に来れるだなんて、\n素直でいい子ですね。\nもちろん見せてあげますよ。ほら…❤,Coming over to surrender of your own volition... what a good obedient boy you are.\nI'd be happy to show you. Now then...❤
```
Remove space or capitalize W in What

Giratena commented on CSV/Language/English/Card/70002.txt:
```diff
@@ -22,16 +22,16 @@ default
-くすくす❤みっともない姿ね。,Kuhuhu-❤ That's not a very distinguished look for you.
-どう？カードの快楽攻撃が、どれほどのものか実感できたかしら。,Well? What do you think? Do you understand cards and pleasure work in battle now? 
-って、あなたはカードの攻撃を受けるまえに、\nひとりでシコシコ始めちゃったからなにも分からないか。くすくす❤,Though; considering you started playing with yourself before you even got attacked...maybe you don't. Kuhuhu-❤
+くすくす❤みっともない姿ね。,Kuhuhu-❤ That's not a very dignified look for you.
```
Good catch!

Giratena commented on CSV/Language/English/Card/70002.txt:
```diff
@@ -72,7 +72,7 @@ default
 あら、すっかりおっぱいに骨抜きね。\n女性モンスターにご奉仕されて、その厄介さが分かったかしら。,Ara; you can feel yourself becoming completely helpless for her boobs; can't you? Perhaps now you understand how troublesome being serviced by a female monster can be. 
 あら、手わざにすっかり骨抜きね。\n女性モンスターにご奉仕されて、その厄介さが分かったかしら。,Ara; you can feel yourself becoming infatuated with her hands; can't you? Perhaps now you understand how troublesome being serviced by a female monster can be. 
-快楽攻撃を受けて集中を乱されたり、\n戦う意思自体を奪われたり、\n力任せだけじゃない強さをもっと教えてあげる❤,Distracted by pleasure attacks...having your will to fight snatched away...I'll show you that there is more to strength than just brute force.❤
+快楽攻撃を受けて集中を乱されたり、\n戦う意思自体を奪われたり、\n力任せだけじゃない強さをもっと教えてあげる❤,Distracted by pleasure attacks... having your will to fight snatched away... I'll show you that there is more to strength than just brute force.❤
```
I think this change should be reverted for reasons already stated ^^

Giratena reviewed (changes requested):
Not bad at all!

I left some comments. Once those are fixed I think we can get this merged!

John-hash-tech committed:
Update 10001.txt

Removed all \n characters from the English text for Succubus Maid except for the one in her profile

John-hash-tech committed:
Update 10003.txt

Removed all \n characters from the English text for Samurai Master Shizuku

John-hash-tech committed:
Update 10001.txt

Changed multiple instances of "in my breasts" to "between my breasts" in the file for Succubus Maid. Left the instance in line 58 unchanged because I like how it's currently written.

John-hash-tech committed:
Update 10003.txt

Fixed a missing word in the file for Samurai Master Shizuku

John-hash-tech committed:
Update 10002.txt

Removed all \n characters from the English text for Samurai Girl

John-hash-tech committed:
Update 10002.txt

Changed one instance of "in my breasts" to "between my breasts" in the file for Samurai Girl.

John-hash-tech committed:
Update 10002.txt

Changed a word in Samurai Girl's English text. It looks kinda silly, but it parallels the "must" earlier in the sentence and I think it fits with her slightly old-timey manner of speech.

John-hash-tech committed:
Update 10002.txt

Removed an extra space in the Samurai Girl file

John-hash-tech committed:
Update 10001.txt

Changed a line in Succubus Maid's English text in response to PR feedback

John-hash-tech committed:
Update 70002.txt

Replaced a Japanese exclamation point with the normal English character equivalent.

John-hash-tech committed:
Update 70001.txt

This line uses both "also" and "as well," which is redundant.

John-hash-tech committed:
Update 70002.txt

Since the monster grounds are plural, we should use "lie" here.

John-hash-tech committed:
Update 70002.txt

Plural possessive apostrophe placement, assuming there is more than one monster in the monster grounds.

John-hash-tech committed:
Update 70002.txt

"Monsters" is plural, so "Succubi" should be as well.

John-hash-tech committed:
Update 70002.txt

Hyphenated "B-Rank" to better match the profile Master Konoha has in the first battle.

John-hash-tech commented on CSV/Language/English/Card/10003.txt:
I notice there are several places in Samurai Master - Shizuku's text where we use hyphens to end sentences. For example, line 7:
`ほら、やわらかいおっぱいですよー❤,Look at how soft my breasts are-❤`
Is that something we want to do?

HotHeadEmperor replied:
If it's before hearts, probably change it to a ~

If there aren't any hearts, use an appropriate punctuation.

This was probably a practice in place that got abandoned, because I haven't seen it in a while. Might have been what was done before the use of ~ was standardized, if I had to guess.

John-hash-tech committed:
Update 10003.txt

Changing him to her, since this line appears to be referring to Master Konoha.

John-hash-tech committed:
Update 10001.txt

Removed a duplicate word from Succubus Maid

John-hash-tech committed:
Update 10001.txt

into -> in to

John-hash-tech committed:
Update 10001.txt

typo

John-hash-tech commented on CSV/Language/English/Card/10001.txt:
I notice line 131 has a rare instance of "Boobjob." It looks like we use "boobjob" 46 times across all files, while "paizuri" is used 225 times. Is this something we want to standardize? This instance is also interesting because it's the selection popup when you beg the Succubus Maid. Most of the other occurrences are probably in dialog, where it makes sense that different terms would be used.

Giratena replied:
I believe it depends on the character. All are valid!

Personality I would go for Paizuri for Succubi, but Boobjob for human cardmasters for example.

John-hash-tech committed:
Update 10003.txt

Punctuation updates

John-hash-tech committed:
Update 10003.txt

Added commas to separate clauses for Samurai Master Shizuku

John-hash-tech committed:
Update 10003.txt

Cleaned up the use of hyphens as sentence-ending punctuation. Since all but one were used before hearts, I used chose to replace them with tildes, which I believe convey the flirtatious tone properly.

John-hash-tech committed:
Update 10003.txt

Changed "hips would give in" to "hips would give out," since they are probably buckling from weakness rather than complying with demands.

John-hash-tech committed:
Update 70002.txt

Changed Master Konoha's bio from the second battle to use a more suitable word.

John-hash-tech committed:
Hora sweep for tutorial

Cleaned up the few instances of "hora" present in the beginning tutorial duels. Since Maid Succubus and Samurai Girl have clearer gimmicks than the slime girls, I tried to go with more thematic interpretations than just "here" or "come on."

I also caught a few lingering em dashes being used as sentence-ending punctuation and made some slight adjustments in a few other spots.

John-hash-tech committed:
Update 40037.txt

Renamed a card that was probably the victim of a mass text change. "Elemental Harmony" is pretty intuitive given the art and effect of the card, but I can also understand the desire to avoid confusing Elements and Attributes.

John-hash-tech committed:
Playtesting hora changes

Updated multiple areas of Samurai Girl's dialog, plus one missing word in Maid Succubus's dialog.

John-hash-tech committed:
Card wording changes

The way card effects are structured should be brought more in line with normal English sentence structure. These changes make an effort to do that, although some discussion will be necessary to decide the overall direction for the project.

John-hash-tech commented on CSV/Language/English/Card/10002.txt:
```diff
@@ -12,8 +12,8 @@ EU2_paizuriAnime,
-童貞マスターのおぬしなら、\nパイズリですぐ喘いでしまうのでござろう？ほらほらほら❤,Being a virgin master...\nIt's only natural that you'd start panting from a simple paizuri; wouldn't you? Hora; hora; hora~❤
-服越しに胸を押し付けてやるだけで、\nすぐ喘いでしまうのでござろう？ほらほらほら❤,Simply pressing my breasts against you through the fabric of my own clothes is already making you start to pant; isn't it? Hora; hora; hora~❤
+童貞マスターのおぬしなら、\nパイズリですぐ喘いでしまうのでござろう？ほらほらほら❤,Being a virgin master... It's only natural that you'd start panting from a simple paizuri; wouldn't you? I'll do; I'll do; and I'll do~❤
+服越しに胸を押し付けてやるだけで、\nすぐ喘いでしまうのでござろう？ほらほらほら❤,Simply pressing my breasts against you through the fabric of my own clothes is already making you start to pant; isn't it? I'll do; I'll do; and I'll do~❤
```
Looking for suggestions here. I tried to get more creative with the hora changes than just "here" or "come on," but this horahorahora was a bit of a stumper. Since Samurai Girl's dialog is old-timey, I borrowed "I'll do, I'll do, and I'll do" from Shakespeare, but I get the feeling the average reader will see this as out of place.

Giratena replied:
Hmm I think it's a very cool reference but sadly a little out of place here.

Usually with Hora its more of a "take this and this and this!" Kind of vibe. 

- [ ] revert back to Hora or localize differently

John-hash-tech committed:
Update 10001.txt

Changed the maid's "if you will" to "if you would." The goal is to make her sound polite, not just "if you X, then I'll Y."

John-hash-tech commented on CSV/Language/English/Card/40037.txt:
```diff
@@ -1,3 +1,3 @@
 default
-エレメントの調和,Attributeal Harmony
+エレメントの調和,Elemental Harmony
```
This looks like a mass text change oversight. I know we're not using "Element" as a keyword in this game, but Yugioh has a precedent of using the word "Element" in card names that care about certain Attributes.

Giratena replied:
I'm ok with that change ^^

John-hash-tech committed:
Update 70002.txt

Comma fixes

John-hash-tech commented:
Ok, ready for review again!

Giratena reviewed (changes requested):
Looking good! Just revert the I do, I do I do change and we can merge this in ^^

John-hash-tech committed:
PR Feedback

Retranslated one horahorahora.

John-hash-tech committed:
PR feedback

Cleaned up unescaped commas

Giratena reviewed (approved):
Looking good!
